server/services/lduService.js

```diff
@@ -13,19 +16,25 @@ const logger = require('../../log')
 module.exports = function createLduService(deliusClient, activeLduClient) {
   return {
     async getAllProbationAreas() {
-      return deliusClient.getAllProbationAreas()
+      const probationAreas = await deliusClient.getAllProbationAreas()
+      sortObjArrayAsc(probationAreas)
+      return probationAreas
     },
 
     async getLdusForProbationArea(probationAreaCode) {
       const allLdus = await deliusClient.getAllLdusForProbationArea(probationAreaCode)
       const activeLdus = await activeLduClient.allActiveLdusInArea(probationAreaCode)
       const activeLdusCodes = activeLdus.map(ldu => ldu.code)
+      const probationAreaDescription = getIn(allLdus, [0, 'description'])
```

Comment:
getAllLdusForProbationArea returns an array of Ldu. Each Ldu has code and description of that LDU. I don't understand how you can get a probationAreaDescription from the first Ldu.
Also, below, since an Ldu does not have a 'teams' field I don't see how you can extract a team from the first element of the allLdus array.  I suspect you've mixed up the 'teams' within one LDU with the list of LDUs.

Probably want to do something like 
allLdus.map(ldu => ({
  code: ldu.code,
  description: ldu.description,
  active: activeLduCodes.includes(ldu.code)
}))

Reply:
The 'teams' came from the obj Delius is returning.... I think ...
I got it from where they had this example

[   {     "code": "N01",     "description": "NPS North West",     "teams": [       {   "code": "string",         "description": "string",    plus more stuff  }]   

plus more stuff
 } ]
--

Reply:
done